OpenRA.Utility/Program.cs

```diff
@@ -62,7 +62,8 @@ static void Main(string[] args)
 			{
 				var command = (IUtilityCommand)Activator.CreateInstance(commandType);
 				var kvp = new KeyValuePair<Action<ModData, string[]>, Func<string[], bool>>(command.Run, command.ValidateArguments);
-				actions.Add(command.Name, kvp);
+				if (!actions.ContainsKey(command.Name))
+					actions.Add(command.Name, kvp);
```

Comment:
Or keep a `--import-map` command, and automatically choose an importer based on the passed mod (as all utility commands require a mod).